### DISCUSSION TITLE
Action bar

### DISCUSSION THREAD

**Discussion by raphael2228** [Adriwin06/Ultimate-CommonUI-Menu-System#13] [Q&A]
It's me again, I'm continuing my crazy adventure on the settings menu with a new problem with action (the bottom bar with the action buttons (save/ restore/ back ect). I can't get the action buttons to appear, my bar is all empty. common ui is brand new to me, I've probably forgotten something and the doc doesn't really help me. apart from that, everything works perfectly and is very easy to adapt, unlike Lyra's, I think.

**Comment by raphael2228** [comment]
![Capture d’écran 2024-06-08 011926](https://github.com/Adriwin06/Ultimate-CommonUI-Menu-System/assets/63161462/536da060-db19-423e-945d-c01a7dcc7471)
![Capture d’écran 2024-06-08 011907](https://github.com/Adriwin06/Ultimate-CommonUI-Menu-System/assets/63161462/36cdb47c-096d-4ca6-99d7-8873190f0aa5)

**Reply by Adriwin06** [reply]
You have to check "Display in Action Bar" in the buttons settings
